<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brightness_contrast_editor</class>
 <widget class="QDialog" name="brightness_contrast_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>B&amp;C Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <widget class="QPushButton" name="reset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="minimum_label">
     <property name="text">
      <string>Minimum:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="plot_layout"/>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="maximum">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="maximum_label">
     <property name="text">
      <string>Maximum:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="max_label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSlider" name="brightness">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="auto_button">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="minimum">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="brightness_label">
     <property name="text">
      <string>Brightness:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="min_label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="contrast_label">
     <property name="text">
      <string>Contrast:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSlider" name="contrast">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="set_data_range">
     <property name="text">
      <string>Set Data Range</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>minimum</tabstop>
  <tabstop>maximum</tabstop>
  <tabstop>brightness</tabstop>
  <tabstop>contrast</tabstop>
  <tabstop>auto_button</tabstop>
  <tabstop>reset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
